### ISSUE TITLE
mpris-notifier stopped working after dbus update

### ISSUE THREAD

**Issue by l1na-forever** [l1na-forever/mpris-notifier#2]
Cutting this to myself. After getting a [dbus update in Void's repos](https://github.com/void-linux/void-packages/commit/b8d0baec401d6b9ec705af97851b199fcbdbac4d), `mpris-notifier` suddenly stopped connecting:

```
$ echo $DBUS_SESSION_BUS_ADDRESS                                                                                                                                                                             
unix:path=/tmp/dbus-28xxxxxxxx,guid=xxxxxxxxx8198ffd295e9c5a5637c20bf
$ mpris-notifier
Error: DBus(Connection(PathDoesNotExist("/tmp/dbus-28xxxxxxxx,guid=xxxxxxxxx8198ffd295e9c5a5637c20bf")))
````

Looking at the [DBus changelog](https://raw.githubusercontent.com/freedesktop/dbus/dbus-1.14/NEWS), there's a related behavior change:

> On Linux, dbus-daemon and other uses of DBusServer now create a
  path-based Unix socket, unix:path=..., when asked to listen on a
  unix:tmpdir=... address. This makes unix:tmpdir=... equivalent to
  unix:dir=... on all platforms.
>  Previous versions would have created an abstract socket, unix:abstract=...,
  in this situation.
 > This change primarily affects the well-known session bus when run via
  dbus-launch(1) or dbus-run-session(1). 

I have a feeling [rustbus' session bus address parsing](https://github.com/KillingSpark/rustbus/blob/master/rustbus/src/connection.rs#L82) doesn't account for a `unix:path` that includes the `,guid=` field.

**Comment by l1na-forever** [comment]
[Pull request opened to rustbus](https://github.com/KillingSpark/rustbus/pull/81), which fixes the issue for me.

**Comment by l1na-forever** [comment]
merged into rustbus, and released as 0.15 here
